plugins/org.python.pydev/tests_completions/org/python/pydev/ast/codecompletion/revisited/ModulesManagerTest.java

```diff
@@ -99,6 +100,7 @@ public void testLoad() throws Exception {
         manager.addModule(new ModulesKey("foo", new File("foo.py")));
         manager.addModule(new ModulesKey("empty", null));
         manager.addModule(new ModulesKeyForZip("zip", new File("zip.zip"), "path", true));
+        manager.addModule(new ModulesKeyForFolder("folder.__init__", new File("folder")));
```

Comment:
I inserted `.__init__` in the ModuleKey name based on how others ModulesKeyForFolder are being stored (same for file path)